kasa/smart/modules/lighteffectmodule.py

```diff
@@ -0,0 +1,112 @@
+"""Module for light effects."""
+
+from __future__ import annotations
+
+import base64
+import copy
+from typing import TYPE_CHECKING, Any
+
+from ...exceptions import KasaException
+from ...feature import Feature
+from ..smartmodule import SmartModule
+
+if TYPE_CHECKING:
+    from ..smartdevice import SmartDevice
+
+
+class LightEffectModule(SmartModule):
+    """Implementation of dynamic light effects."""
+
+    REQUIRED_COMPONENT = "light_effect"
+    QUERY_GETTER_NAME = "get_dynamic_light_effect_rules"
+    AVAILABLE_BULB_EFFECTS = {
+        "L1": "Party",
+        "L2": "Relax",
+    }
+    LIGHT_EFFECTS_OFF = "Off"
+
+    def __init__(self, device: SmartDevice, module: str):
+        super().__init__(device, module)
+        self._scenes: dict[str, str] = {}
+
+    def _initialize_features(self):
+        """Initialize features."""
+        device = self._device
+        self._add_feature(
+            Feature(
+                device,
+                "Light effect theme",
```

Comment:
```suggestion
                "Light effect",
```

Reply:
In the TAPO app for the bulbs this is called "Theme", there is no mention of "effects" (unlike the smartstrip where it is called "effects").  I think we should keep "theme" included so as not to confuse.

Reply:
I'd say keep it simple for now, like just "Effect" as that's what the kasa devices use for non-custom ones which would map directly to the light effect feature on homeassistant? Although it's not a big deal, so your call!

Reply:
Done